package.json

```diff
@@ -63,7 +63,8 @@
     "update:sample-json": "node ./lighthouse-cli -A=./lighthouse-core/test/results/artifacts --throttling-method=devtools --output=json --output-path=./lighthouse-core/test/results/sample_v2.json http://localhost/dobetterweb/dbw_tester.html && node lighthouse-core/scripts/cleanup-LHR-for-diff.js ./lighthouse-core/test/results/sample_v2.json --only-remove-timing",
     "diff:sample-json": "bash lighthouse-core/scripts/assert-golden-lhr-unchanged.sh",
     "update:crdp-typings": "node lighthouse-core/scripts/extract-crdp-mapping.js",
-    "mixed-content": "./lighthouse-cli/index.js --chrome-flags='--headless' --config-path=./lighthouse-core/config/mixed-content.js"
+    "mixed-content": "./lighthouse-cli/index.js --chrome-flags='--headless' --config-path=./lighthouse-core/config/mixed-content.js",
+    "minify-latest-run": "./lighthouse-core/scripts/lantern/minify-trace.js ./latest-run/defaultPass.trace.json ./latest-run/defaultPass.trace.json && ./lighthouse-core/scripts/lantern/minify-devtoolslog.js ./latest-run/defaultPass.devtoolslog.json ./latest-run/defaultPass.devtoolslog.json"
```

Comment:
latest-run/defaultPass.trace.json is in there twice. same with the dtlog.

Reply:
if it's going to do it in place, maybe make the default output just be the input? Would make it a lot easier to read here, at least (otoh you might be sad that it automatically wrote over your saved trace, but I don't foresee many folks running this script independently)

Reply:
I'd prefer not to make the default mode be inplace, I can put the latest-run files into different files? `defaultPass.minified.trace.json`?

Reply:
that's fine (either way, really). It was mostly a comment on npm scripts being generally unreadable and this with the doubled-up everything is especially hard to read :)